giotto/graphs/__init__.py

```diff
@@ -5,9 +5,18 @@
 from giotto.graphs.geodesic_distance import GraphGeodesicDistance
 from giotto.graphs.kneighbors import KNeighborsGraph
 from giotto.graphs.transition import TransitionGraph
+from giotto.graphs.create_clique_complex import CreateCliqueComplex, \
+    CreateBoundaryMatrices, CreateLaplacianMatrices
```

Comment:
According to the sklearn conventions, all the imports from modules that reside inside the library (in this case inside giotto) should be relative (e.g. 'from .graphs.transition import TransitionGraph). 
Absolute imports should instead be use for test packages.